index.yaml

```diff
@@ -57,6 +57,13 @@ indexes:
 
 - kind: ExplorationCommitLogEntryModel
   properties:
+  - name: deleted
```

Comment:
I don't see any additional calls to ExplorationCommitLogEntryModel in this PR. Why is the additional index entry needed?